Dockerfile

```diff
@@ -1,28 +1,19 @@
 ###############################################################
 ### ---- Create a Docker image for Jolie Documentation ---- ###
 ###############################################################
-FROM nginx:alpine
-LABEL build_date="2020-08-31" \
-	  author="Brian Alberg <alberg@imada.sdu.dk>" \
-	  description="Jolie Documentation" \
-	  url="https://jolie-lang.org"
-MAINTAINER Brian Alberg "alberg@imada.sdu.dk"
-ENV REFRESHED_AT 2020-08-31
-## Install NodeJS
-RUN apk update && \
-	apk add --update nodejs npm --no-cache
-WORKDIR /home/jolie-docs
-COPY gitbook/* ./
-# sed append line after match
-RUN sed -i '/^{/a \    "root": "./web",' book.json
-RUN npm update
+FROM node:alpine
 RUN npm install -g gitbook-cli
-RUN npm install mv
-RUN npm install gitbook-plugin-highlight-jolie
-RUN npm install gitbook-plugin-theme-jolie
-RUN npm install gitbook-plugin-logo
-RUN npm install gitbook-plugin-collapsible-chapters
```

Comment:
By removing these plugins we don't have the syntax-highlighting, theme or logo anymore. Did you move these somewhere else?

Reply:
Those plugins are automatically installed by `gitbook install`.